tools/synchro.py

```diff
@@ -148,6 +148,7 @@
     "test_collection.py",
     "test_database.py",
     "test_session.py",
+    "test_transactions.py",
```

Comment:
Mistaken addition or necessary to make synchro tests pass?

Reply:
Merge artifact, fixed.